compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -365,12 +365,12 @@ OMR::X86::CodeGenerator::initialize(TR::Compilation *comp)
    self()->setSupportsArrayCmp();
    self()->setSupportsArrayCopy();
 
-   if (comp->getOption(TR_EnableX86AdvancedMemorySet))
-      self()->setSupportsArraySet();
-
    static char *disableArraySet = feGetEnv("TR_disableArraySet");
```

Comment:
Any particular reason this is an env var and not an option check ? Options can be applied per compiled method whereas env vars cannot.